internal/tests/cluster/x509_verification_test.go

```diff
@@ -185,13 +186,22 @@ func TestCustomX509Verification_Client(t *testing.T) {
 }
 
 func TestCustomX509Verification_Server(t *testing.T) {
+	t.Skip("These tests are currently not working and will need further investigation")
+
 	ec := event.TestEventerConfig(t, "TestCustomX509Verification_Server", event.TestWithObservationSink(t), event.TestWithSysSink(t))
 	testLock := &sync.Mutex{}
 	logger := hclog.New(&hclog.LoggerOptions{
 		Mutex: testLock,
 		Name:  "test",
+		Level: hclog.Trace,
 	})
 	require.NoError(t, event.InitSysEventer(logger, testLock, "use-TestCustomX509Verification_Server", event.WithEventerConfig(&ec.EventerConfig)))
+	t.Cleanup(func() { event.TestResetSystEventer(t) })
+	t.Cleanup(func() {
+		all, err := io.ReadAll(ec.AllEvents)
+		require.NoError(t, err)
+		t.Log(string(all))
+	})
```

Comment:
Is the purpose of having multiple cleanups back-to-back is just to group like things together?

Reply:
I see them as distinct cleanups, one is reading the output of the events at the end of the test, and the other one is resetting the syseventer (the inverse of the line above it). Putting them in one cleanup is mixing concerns a bit IMO, but it's purely a matter of style, as long as the read runs before the resetter.